kosmorrolib/enum.py

```diff
@@ -25,7 +25,10 @@ class EventType(Enum):
     MAXIMAL_ELONGATION = 4
     MOON_PERIGEE = 5
     MOON_APOGEE = 6
-
+    VERNAL_EQUINOX = 7
+    SUMMER_SOLSTICE = 8
+    AUTUMNAL_EQUINOX = 9
+    WINTER_SOLSTICE = 10
```

Comment:
We should use more neutral names here, since the seasons are inverted in the South hemisphere (e.g. on June 20th, it is Summer in North hemisphere and Winter in South hemisphere).

We could use the same model as the `SEASON_EVENTS_NEUTRAL` of [Skyfield's Almanac](https://github.com/skyfielders/python-skyfield/blob/master/skyfield/almanac.py#L65-L70), with the months names instead of the seasons'.